### PR TITLE
Add general commands doc

### PR DESCRIPTION
**Pull request by akhilkumarpilli**
Help docs 
- Send Tokens
- Query Balances
- Delegate (stake)
- Undelegate
- Redelegate
- Withdraw rewards
- Query Rewards
- Recover Keys

### CODE REVIEW

File: usage/cli/general-commands.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+### Send Tokens
+
+Sign and send tokens to other account.
+>**Usage**
+
+`akash tx send <from-account> <to-account> <amount-with-denom>`
+> **Example**
+```
+$ akash tx send bob akash12wlzqjelrt87r3u56ps4m8lk7wavx5m5jg9cax 10000000stake
```
Comment (anilCSE) [inline usage/cli/general-commands.md]:
```suggestion
$ akash tx send bob akash12wlzqjelrt87r3u56ps4m8lk7wavx5m5jg9cax 10000000stake -o json
```

File: usage/cli/general-commands.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+### Bond(Stake) Tokens
+Delegate some tokens to validator i.e., bonding tokens.
+> **Usage**
+```
+$ akash tx staking delegate <validator-address> <amout-with-denom> --from <from-account>
+```
+> **Example**
+```
+$ akash tx staking delegate akashvaloper1qgjla78dk2yk2eh6dmjeejz6m374ykdtpgp92s 1000000stake --from alice
```
Comment (anilCSE) [inline usage/cli/general-commands.md]:
```suggestion
$ akash tx staking delegate akashvaloper1qgjla78dk2yk2eh6dmjeejz6m374ykdtpgp92s 1000000stake --from alice -y -o json
```

**Comment by anilCSE** [comment]
@akhilkumarpilli can you also add `query tx`?

**Comment by gosuri** [comment]
This is great!

**Pull request merged**